library/ssl_debug_helpers.h

```diff
@@ -1,4 +1,3 @@
-/* Automatically generated by generate_ssl_debug_helpers.py. DO NOT EDIT. */
```

Comment:
We probably want the usual copyright and license notice, though.

Reply:
That and the lack of documentation are preexisting and I wanted to make this fix as quick as possible. But ok, I've added the license. I'll pass on documentation.